tests/phpunit/api/v3/ContactTest.php

```diff
@@ -116,6 +117,29 @@ public function testAddCreateIndividual() {
   }
 
   /**
+   * Test that it is possible to prevent cache clearing via option.
+   *
+   * Cache clearing is bypassed if 'options' => array('do_not_reset_cache' => 1 is used.
+   */
+  public function testCreateIndividualNoCacheClear() {
+
+    $contact = $this->callAPISuccess('contact', 'create', $this->_params);
+    $groupID = $this->groupCreate();
```

Comment:
This isn't getting cleaned up in `tearDown()` which causes the next test to use this method to fail because the group already exists.